### PR TITLE
osx

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
pending https://github.com/conda-forge/fabio-feedstock/pull/5

**Comment by CJ-Wright** [comment]
@conda-forge/pyfai ready for review 
special attention to @chiahaoliu since you have a mac.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,8 +13,9 @@ source:
 
 build:
   number: 0
-  script: python setup.py install --record record.txt
-  skip: True  # [not linux]
+  script: python setup.py install --record record.txt  # [linux]
+  script: rm pyFAI/ext/histogram.c & python setup.py install --no-openmp  # [osx]
```
Comment (tacaswell) [inline recipe/meta.yaml:17]: Why nuke `histogram.c`?  Is this a non-critical file

Comment (CJ-Wright) [reply recipe/meta.yaml:17]: pyFAI said to https://github.com/silx-kit/pyFAI#macosx

**Comment by CJ-Wright** [comment]
@tacaswell Can I get a second round?

**Pull request merged**